content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
+:guilabel:`Internal Reference` (a customizable product reference idenifier in Odoo, like `FURN001`)
+with the Amazon *SKU* for marketplace items, the Amazon *Shipping Code* for delivery charges, and
+the Amazon *Gift Wrapping* code for gift wrapping charges.
+
+For marketplace products, pairings are saved as *Amazon Offers*, which are listed under the
+:guilabel:`Offers` smart button on the account form.
 
-For marketplace products, pairings are saved as **Amazon Offers** which are listed under the
-**Offers** stat button on the account form. Offers are automatically created when the pairing is
-established and are used for subsequent orders to lookup SKUs. If no offer with a matching SKU is
-found, :ref:`the internal reference is used instead <amazon/matching>`.
+.. image:: setup/amazon-offers-button.png
+   :align: center
+   :alt: The Amazon Offers smart button on the account form in Odoo Sales.
+
+Offers are automatically created when the pairing is established, and they're used for subsequent
+orders to lookup SKUs. If no offer with a matching SKU is found, :ref:`the internal reference is
+used instead <amazon/matching>`.
 
 .. tip::
-   It is possible to force the pairing of a marketplace item with a specific product by changing
-   either the product or the SKU of an offer. The offer can be manually created if it was not
-   automatically done yet. This is useful if you do not use the internal reference as the SKU or if
-   you sell the product under different conditions.
+   It's possible to force the pairing of a marketplace item with a specific product, by changing
+   either the product or the SKU of an offer to ensure they match. The offer can be manually created
+   if it was not automatically done yet.
+
+   This is useful if the internal reference is not used as the SKU, or if the product sells under
+   different conditions.
 
-If no database product with a matching internal reference is found for a given SKU or gift wrapping
-code, a default database product **Amazon Sale** is used. The same is done with the default product
-**Amazon Shipping** and the shipping code.
+If no database product with a matching internal reference is found for a given Amazon SKU or gift
+wrapping code, then a default database product, *Amazon Sale*, is used. The same is done with the
+default product *Amazon Shipping* if no database product is found for a given Amazon shipping code.
 
 .. note::
-   To modify the default products, activate the :ref:`developer mode <developer-mode>` and navigate
-   to :menuselection:`Sales --> Configuration --> Settings --> Connectors --> Amazon Sync -->
+   To modify default products, activate the :ref:`developer mode <developer-mode>`, and navigate to
+   :menuselection:`Sales app --> Configuration --> Settings --> Connectors --> Amazon Sync -->
    Default Products`.
 
-Configure taxes of products
-===========================
+Product tax configuration
```

Comment:
There's no actual instruction here in this section.

Ideally there'd be links to Accounting articles, as well, to further help the reader in this use case, e.g. setting taxes on a product form or determining a fiscal position.